Sitronix/GHIElectronics.TinyCLR.Drivers.Sitronix.ST7735/ST7735.cs

```diff
@@ -1,5 +1,222 @@
-﻿namespace GHIElectronics.TinyCLR.Drivers.Sitronix.ST7735 {
+﻿using System.Threading;
+
+using GHIElectronics.TinyCLR.Devices.Gpio;
+using GHIElectronics.TinyCLR.Devices.Spi;
+
+namespace GHIElectronics.TinyCLR.Drivers.Sitronix.ST7735 {
+    public enum ColorFormat {
+        BGR12bit444,
+        BGR16bit565
+    }
+
     public class ST7735 {
+        private const byte ST7735_MADCTL = 0x36;
+        private const byte MADCTL_MY = 0x80;
+        private const byte MADCTL_MX = 0x40;
+        private const byte MADCTL_MV = 0x20;
+        private const byte MADCTL_BGR = 0x08;
+
+        private readonly SpiController spiBusContoller;
+        private readonly SpiDevice spiBus;
+
+        private readonly GpioPin resetPin;
+        private readonly GpioPin controlPin;
+
+        private readonly byte[] singleByteBuffer;
```

Comment:
The name for something like this that we usually follow is `buffer1`, `buffer2`, `buffer4`, etc

Reply:
I will change it to buffer1 but I think singleByteBuffer, while wordier, is more concise.